.github/workflows/maven.yml

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Build OB Enterprise Repository
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches:
+      - 'master'
+    types:
+      - labeled
+
+jobs:
+
+  Build-repo-check:
+
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'Ready-to-Build') }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
```

Comment:
Shall we have the workflow configured in Java 11?

Reply:
fixed with https://github.com/wso2/financial-open-banking/commit/61546667167a516b5099c60c9dd0eb25bea13bac